addon/styles/addon.css

```
@@ -70,3 +70,21 @@
   color: red;
   font-size: large;
 }
+
+.closing-bracket {
+  display: inline-block;
+}
+
+pre {
+  display: inline;
+  padding: 0px;
```

```suggestion
  padding: 0;
```
0 doesn't need units (since it is 0 of whatever unit you would use... 0ems is 0px is 0cm)